README.md

```diff
@@ -125,5 +159,74 @@ pip install --editable .
 ```
 
 
-## Credit
+## Deployment
+
+### Requirements
+
+*Instructions for AWS Linux 2023*
+
+1. **Python3.11**: Install with `sudo yum install python3.11`
+2. **git**: Install with `sudo yum install git`
+3. **draft** (this package):
+    1. Clone with `git clone https://github.com/private-attribution/draft.git`
+    2. Enter directory `cd draft`.
+    3. Create virtualenv: `python3.11 -m venv .venv`
+    4. Use virtualeenv: `source .venv/bin/activate`
+    5. Upgrade pip: `pip install --upgrade pip`
+    6. Install: `pip install --editable .`
+4. **traefik**:
+    1. Download version 2.11: `wget https://github.com/traefik/traefik/releases/download/v2.11.0/traefik_v2.11.0_linux_amd64.tar.gz`
+    2. Validate checksum: `sha256sum traefik_v2.11.0_linux_amd64.tar.gz` should print `7f31f1cc566bd094f038579fc36e354fd545cf899523eb507c3cfcbbdb8b9552  traefik_v2.11.0_linux_amd64.tar.gz`
+    3. Extract the binary: `tar -zxvf traefik_v2.11.0_linux_amd64.tar.gz`
+5. **tmux**: `sudo yum install tmux`
+
+
+### Generating TLS certs with Let's Encrypt
```

Comment:
this doesn't invoke sudo within the virtualenv, it just references the binary originally installed within the virtualenv (to avoid polluting the system wide python packages.)